specs/BrowserTaskManager.md

```diff
@@ -0,0 +1,81 @@
+# Background
+The WebView2 team has been asked to provide an API to expose the Browser Task
+Manager. The Browser Task Manager is a window helpful for debugging that 
+displays the different processes associated with the current browser process and 
+what they're used for. For instance, it could denote that a particular process 
+is a renderer process and would show the different web pages rendered by that 
+process. In a chromium browser it can be opened by the end user by pressing 
+`Shift+Esc` or from the DevTools window's title bar's context menu entry 
+`Browser task manager`. For WebView2 applications, we block the 
+`Shift+Esc` shortcut, but the task manager can still be opened via the 
+`Browser task manager` entry.
```

Comment:
Curious: Can the app disable the DevTools window?

Reply:
Yes, there's a setting to disable the end user UI entry points to the DevTools window.